Formula/imap-uw.rb

```diff
@@ -45,7 +52,16 @@ def install
     end
     inreplace "src/osdep/unix/ssl_unix.c", "#include <x509v3.h>\n#include <ssl.h>",
                                            "#include <ssl.h>\n#include <x509v3.h>"
-    system "make", "oxp"
+    on_macos do
+      system "make", "oxp"
+    end
+
+    on_linux do
+      # Skip IPv6 warning on Linux as libc should be IPv6 safe.
+      touch "ip6"
+
+      system "make", "ldb"
+    end
```

Comment:
Does this work?
```suggestion
    # Skip IPv6 warning on Linux as libc should be IPv6 safe.
    touch "ip6"

    target = "oxp"
    on_linux { target = "ldb" }
    system "make", target
```

Reply:
Yes, it should work. Applied the suggestions.